driver/csiplugin/nodeserver.go

```diff
@@ -117,8 +120,9 @@ func getGpfsPaths(ctx context.Context) []string {
 			finalOutput := strings.Split(out, " ")
 			if len(finalOutput) == mountPathLength {
 				if finalOutput[1] != "" && finalOutput[2] == "gpfs" {
-					val, ok := os.LookupEnv(isOpenShiftCluster)
-					if ok && val == "True" {
+					configurationType, ok := os.LookupEnv(ENVCluster_ConfigurationType)
+					cnsaPresence, okk := os.LookupEnv(ENVCluster_CNSAPresenceCheck)
+					if (ok && configurationType == ENVCluster_Type_Openshift) && (okk && cnsaPresence == "True") {
```

Comment:
Only CNSA condition should be used here, please test it once with a pod for all 3 platforms

Reply:
Is this tested on all 3 configs and working fine for a pod mount?

Reply:
In all the 3 env, volume provisioning is working fine.

Reply:
Volume provision has been verified along with volume mount. For the refrence, can be used with below yaml specs.
```
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
   name: ibm-spectrum-scale-csi-fileset
provisioner: spectrumscale.csi.ibm.com
parameters:
    volBackendFs: "fs1"
reclaimPolicy: Delete

---
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: scale-fset-pvc
spec:
  accessModes:
  - ReadWriteMany
  resources:
    requests:
      storage: 1Gi
  storageClassName: ibm-spectrum-scale-csi-fileset

---
apiVersion: v1
kind: Pod
metadata:
  name: csi-scale-fsetdemo-pod
  labels:
    app: alpine
spec:
  containers:
   - name: web-server
     image: quay.io/badri_pathak/nginx:latest
     volumeMounts:
       - name: mypvc
         mountPath: /usr/share/nginx/html/scale
  volumes:
   - name: mypvc
     persistentVolumeClaim:
       claimName: scale-fset-pvc
       readOnly: false
```